awx/api/serializers.py

```diff
@@ -3965,7 +4089,8 @@ def _local_summarizable_fk_fields(self):
         field_list += [
             ('workflow_job_template_node', ('id', 'unified_job_template_id')),
             ('label', ('id', 'name', 'organization_id')),
-            ('notification', ('id', 'status', 'notification_type', 'notification_template_id'))
+            ('notification', ('id', 'status', 'notification_type', 'notification_template_id')),
+            ('access_token', ('id', 'token'))
```

Comment:
Consider tokens the same as passwords; I wouldn't log the token in the activity stream.